libbeat/publisher/queue/proxy/diagrams/broker.d2

```diff
@@ -0,0 +1,54 @@
+# A diagram of the Beats pipeline and the proxy queue's interaction
+# with it.
+# To regenerate the image after changing this file, run:
+#   d2 broker.d2 broker.svg
+# To live-edit this file with immediate regeneration of the diagram, run:
+#   d2 --watch broker.d2 broker.svg
+
+Input
+
+producer {
+    ackHandler
+}
+
+queue: Proxy Queue (broker) {
+    pushChan
+    getChan
+}
+
+Input -> producer: Publish
+producer.ackHandler -> Input: Event acknowledgment
+producer -> queue.pushChan: pushRequest
+queue.pushChan -> producer: result
+
+queueReader {
+    explanation: |md
+        `queueReader` is a worker that reads raw batches (satisfying the<br>
```

Comment:
```suggestion
        `queueReader` is a worker that reads raw batches (satisfying the  
```

replace "\<br\>" with 2 spaces at end of line.  Annoying, but neither github or firefox will render the svg that is produced with "\<br\>".  Need to do for all the "\<br\>" tags.

Reply:
`\` also works btw (https://commonmark.org/help/tutorial/03-paragraphs.html#:~:text=For%20a%20line%20break%2C%20add,the%20end%20of%20the%20line.)

or `<br />` (they won't render it because <br> is not semantic xml)